src/components/reducer.js

```diff
@@ -0,0 +1,67 @@
+import { defaultFilm } from '../constants/const';
+
```

Comment:
A point to think about, perhaps it makes sense to create a constant with all types and, if necessary, simply import it in the future.
```suggestion
export const dispatcherTypes = {
    DELETE: 'delete',
    SUBMIT: 'submit'
    ...
}

import dispatcherTypes from 'sm-th'

const { DELETE } = dispatcherTypes;
...

const onDelete = () => dispatch({type: DELETE}); //If you change the type name, it will be easy because you can click shift + f6 and change a name for a variable or change value.
```

Reply:
done